working-groups/diagnostics/README.md

```diff
@@ -0,0 +1,43 @@
+# Diagnostics Working Group
+![working group status: active][status]
+
+This working group aims to make rustc better at telling the user why the compiler isn't smart enough
+yet to understand their code.
```

Comment:
I think this would read better as “...why the compiler isn’t smart enough to understand their code yet.” 

(I can’t make a suggestion as the sentence is split over two lines.)

Reply:
I would like to add something along the lines of "We attempt to make `rustc` understand a super set of valid Rust code in order to interpret _intent_ code that would be reasonable when extrapolating from valid Rust code, but that isn't valid."